commons/addons/cellar.md

```diff
@@ -115,9 +115,8 @@ You only need to specify a custom endpoint (eg `cellar-c2.services.clever-cloud.
 const AWS = require('aws-sdk');
 
 AWS.config.update({accessKeyId: '<cellar_key_id>', secretAccessKey: '<cellar_key_secret>'});
-const endpoint = new AWS.Endpoint('<cellar_host>');
 
-const s3 = new AWS.S3({ endpoint });
+const s3 = new AWS.S3({ '<cellar_host>' });
```

Comment:
something appears to be missing, I am assuming it's the json key with cellar_host being the value.

Reply:
yes, I missed the key. 
my bad ! it's fixed

Reply:
Thank you !